.gitignore

```diff
@@ -2,3 +2,4 @@
 vendor
 composer.phar
 composer.lock
+.idea/*
```

Comment:
This should be in a global gitignore, not project specific (otherwise we would need eclipse, netbeans, and whatever other ides / editors, which is quite long and not useful)